shared/js/background/events.es6.js

```diff
@@ -629,7 +629,7 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
 let sessionKey = getHash()
 
 function getArgumentsObject (tabId, sender, documentUrl) {
-    const tab = tabManager.get({ tabId })
+    const tab = tabManager.createOrUpdateTab(tabId, sender.tab)
```

Comment:
Can we write down which cases this is for? Is there a case when a content-script message comes before `chrome.tabs.onUpdated` fires? Or is this just for the case when the extension is loaded/reload and content-scripts trigger for existing tabs (in which case we could do `chrome.tabs.query` on startup)?